app/controllers/health_cards_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
   end
 
   def chunks
-    render json: health_card.chunks.to_json
+    render json: HealthCards::Chunking.generate_qr_chunks(jws.to_s)
```

Comment:
I think it would make sense to add this code to jws.rb instead of calling the module directly.

Reply:
Moved it to `jws.rb`